src/storage/mvcc/txn.rs

```diff
@@ -223,7 +322,7 @@ impl<S: Snapshot> MvccTxn<S> {
                         Err(Error::TxnLockNotFound {
                             start_ts: self.start_ts,
                             commit_ts,
-                            key: key.as_encoded().to_owned(),
+                            key: key.to_raw()?,
```

Comment:
Why to_raw?

Reply:
Is it a bug in master branch?

Reply:
Is `into_raw()` better? We can change all `to_raw()` to `into_raw()` if the key will be dropped.